### PR TITLE
[DEV APPROVED] - Syndicate employer best practices

### PR DESCRIPTION
**Pull request by tomas-stefano**
This PR adds employer best practices to syndicated tools to the site.

Add to the test harness will make testing syndication be more easier.

### CODE REVIEW

**Comment by jriga** [comment]
👍

File: widgets/employer_best_practices_cy.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: widget
+widget:
+  id: employer_best_practices
+  name: "Employer Best Practices (Welsh)"
+  url: "/en/employer-best-practices/money-guide"
```
Comment (jones-agyemang) [inline widgets/employer_best_practices_cy.html:6]: The locale is incorrect. 

See here (https://www.moneyadviceservice.org.uk/cy/employer-best-practices/money-guide)

**Comment by margOnline** [comment]
👍

**Comment by tomas-stefano** [comment]
Thank you both! 🙏

**Review by jones-agyemang — COMMENTED** [review]
@tomas-stefano the URL for the Welsh version of the Employee best practice is incorrect.

**Pull request merged**